shell/platform/embedder/embedder_engine.cc

```diff
@@ -146,4 +146,47 @@ bool EmbedderEngine::MarkTextureFrameAvailable(int64_t texture) {
   return true;
 }
 
+bool EmbedderEngine::SetSemanticsEnabled(bool enabled) {
+  if (!IsValid()) {
+    return false;
+  }
+  shell_->GetTaskRunners().GetUITaskRunner()->PostTask(
+      [engine = shell_->GetEngine(), enabled] {
+        if (engine) {
+          engine->SetSemanticsEnabled(enabled);
+        }
+      });
+  return true;
+}
+
+bool EmbedderEngine::SetAccessibilityFeatures(int32_t flags) {
+  if (!IsValid()) {
+    return false;
+  }
+  shell_->GetTaskRunners().GetUITaskRunner()->PostTask(
+      [engine = shell_->GetEngine(), flags] {
+        if (engine) {
+          engine->SetAccessibilityFeatures(flags);
+        }
+      });
+  return true;
+}
+
+bool EmbedderEngine::DispatchSemanticsAction(int id,
+                                             blink::SemanticsAction action,
+                                             std::vector<uint8_t> args) {
+  if (!IsValid()) {
+    return false;
+  }
+  shell_->GetTaskRunners().GetUITaskRunner()->PostTask(
+      [engine = shell_->GetEngine(), id, action = std::move(action),
+       args = std::move(args)] {
```

Comment:
If you are attempting to prevent a copy of the args vector, then this code does not avoid the same. You can use fml::MakeCopyable with a mutable lambda to achieve it though.

Reply:
Done.